test/util/util.go

```diff
@@ -736,3 +736,17 @@ func NewTestingLogger(writer io.Writer, level int) logr.Logger {
 func WaitForNextSecondAfterCreation(obj client.Object) {
 	time.Sleep(time.Until(obj.GetCreationTimestamp().Add(time.Second)))
 }
+
+func ExpectClusterQueuesToBeActive(ctx context.Context, c client.Client, cqs ...*kueue.ClusterQueue) {
+	gomega.EventuallyWithOffset(1, func(g gomega.Gomega) {
+		readCq := &kueue.ClusterQueue{}
+		var inactive []string
+		for _, cq := range cqs {
+			err := c.Get(ctx, client.ObjectKeyFromObject(cq), readCq)
+			if err != nil || !apimeta.IsStatusConditionTrue(readCq.Status.Conditions, kueue.ClusterQueueActive) {
```

Comment:
```suggestion
			g.Expect(err).Should(gomega.Succeeded())
			if !apimeta.IsStatusConditionTrue(readCq.Status.Conditions, kueue.ClusterQueueActive) {
```

Shouldn't we verify if the requested CQ exists in advance?

Reply:
I'ts more a question about what we want the result to be:
1. a full list of cqs for which we cannot say their are active (have condition status !True, or we ware not able to get) 
2. the cause of the fist failure

I've changed it to go with 2.